fractions/teiid/odata/src/main/java/org/wildfly/swarm/odata/ODataFraction.java

```diff
@@ -15,17 +15,23 @@
  */
 package org.wildfly.swarm.odata;
 
+import static org.wildfly.swarm.spi.api.Defaultable.bool;
 import static org.wildfly.swarm.spi.api.Defaultable.string;
 
 import org.wildfly.swarm.spi.api.Defaultable;
 import org.wildfly.swarm.spi.api.Fraction;
 import org.wildfly.swarm.spi.api.annotations.Configurable;
 
 public class ODataFraction implements Fraction<ODataFraction> {
-
-    @Configurable("swarm.odata.context")
+    @Configurable("swarm.teiid.odata.context")
```

Comment:
Might want to keep the original one for compatibility? (But add it as a _2nd_ annotation, as the 1st one is what is documented.)

Reply:
So, I would able to do two @Configurable annotations? Also I willing to break the compatibility for sake of correcting it.

Reply:
I don't know how many users of this fraction is out there, but if you're willing to break compat, then fine :-)

Reply:
I don't think many yet, we are primary consumers of this in another OpenShift based UI project.